packages/zosfiles/src/doc/IOptions.ts

```diff
@@ -18,7 +18,7 @@ import { IZosFilesOptions } from "./IZosFilesOptions";
  * @interface IOptions
  */
 export interface IOptions extends IZosFilesOptions {
-    /**
+    /*
```

Comment:
This change looks like it may have been accidental since JsDoc typically starts with `/**` 🙂